e2e/tests/121-Portal/MakeNewPayment/OCW/PauseCardOCW.spec.ts

```diff
@@ -0,0 +1,58 @@
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import NavigationModule from '@121-e2e/pages/Navigation/NavigationModule';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import NLRCProgram from '@121-service/seed-data/program/program-nlrc-ocw.json';
+import { WalletCardStatus121 } from '@121-service/src/payments/fsp-integration/intersolve-visa/enum/wallet-status-121.enum';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { seedPaidRegistrations } from '@121-service/test/helpers/registration.helper';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
+import { test } from '@playwright/test';
+import englishTranslations from '../../../../../interfaces/Portal/src/assets/i18n/en.json';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdOCW = 3;
+  const OcwProgramId = programIdOCW;
+
+  await seedPaidRegistrations(registrationsOCW, OcwProgramId);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/login');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[28441] Pause Visa debit cards', async ({ page }) => {
```

Comment:
Does it make sense to also add an unpause test? I think you can largely reuse what you already have.

Reply:
It does indeed but I would like to add it in another PR.

Reply:
I added it as an extra step to validate also the unpause